.github/workflows/build-ios-release-pullrequest.yml

```
@@ -18,14 +18,42 @@ jobs:
     env:
       APPLE_ID: ${{ secrets.APPLE_ID }}
+
     steps:
       - name: Checkout Project
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0 # Fetches all history
+          fetch-depth: 0 # Ensures the full Git history is available
+
+      - name: Ensure Correct Branch
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:20: Double quote to prevent globbing and word splitting

(shellcheck)